### PR TITLE
AD-4 created blog app

### PR DESCRIPTION
**Pull request by MrBorusLee**
- created blog app
- created models

### CODE REVIEW

File: adventures/articles/models.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.db import models
+
+
+class Article(models.Model):
+    text = models.TextField()
\ No newline at end of file
```
Comment (MsBorusLee) [inline adventures/articles/models.py]: Is it all needed fields?

Comment (MsBorusLee) [reply adventures/articles/models.py]: oh ok

Comment (MsBorusLee) [reply adventures/articles/models.py]: oh ok

File: adventures/articles/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (MsBorusLee) [inline adventures/articles/tests.py]: What about tests?

Comment (MrBorusLee) [reply adventures/articles/tests.py]: will do

**Pull request merged**